packages/core/src/app/common/error/SentryErrorLogger.ts

```diff
@@ -68,9 +80,15 @@ export default class SentryErrorLogger implements ErrorLogger {
                 new RewriteFrames({
                     iteratee: this.handleRewriteFrame,
                 }),
+                ...defaultIntegrations.filter(integration =>
+                    !['GlobalHandlers', 'RewriteFrames'].includes(integration.name)
+                ),
             ],
-            ...config,
-        });
+            transport: 'fetch' in window ? makeFetchTransport : makeXHRTransport,
+            stackParser: defaultStackParser,
+        };
+
+        this.hub = createHub(clientOptions);
```

Comment:
Hi @bigcommerce/team-checkout , I'm going to revert this PR because this line is not working in production and stopping Sentry from logging any issue.

An example of a production error message:
```
Refused to create a worker from 'blob:https://checkout.solostove.com/5837bd07-3018-47d1-8af1-50e16ed86d04' because it violates the following Content Security Policy directive: "script-src 'none'". Note that 'worker-src' was not explicitly set, so 'script-src' is used as a fallback.
```

<img width="600" alt="Screenshot 2023-12-27 at 11 29 13 am" src="https://github.com/bigcommerce/checkout-js/assets/88361607/ebd14550-78e5-4fa8-b472-707f0a56ac96">

Reply:
After more thorough testing and research, I found out that the proposed solution won’t work. Although Sentry SDK allows multiple client instances for different accounts, many of its features (a.k.a. integrations) only works with the primary client. For example, only the primary client includes breadcrumbs in its error logs, and reports uncaught exceptions. It also won’t work if other client instances are constructed using a different SDK version. This is because all of the client instances actually share the same underlying `__SENTRY__` global variable. Several issues have been reported regarding the limitation of running multiple client instances (e.g.: https://github.com/getsentry/sentry-javascript/issues/3271, https://github.com/getsentry/sentry-javascript/issues/3268, https://github.com/getsentry/sentry-javascript/issues/2732 etc…). Until Sentry SDK has a better support for multiple client instances, I think the only way around this problem is to recommend merchants to not to initialise their own client instance on the checkout page if they are using the default OOPC.